vhost_user_net/src/lib.rs

```diff
@@ -185,7 +185,9 @@ impl VhostUserBackend for VhostUserNetBackend {
     }
 
     fn protocol_features(&self) -> VhostUserProtocolFeatures {
-        VhostUserProtocolFeatures::all()
+        VhostUserProtocolFeatures::MQ
+            | VhostUserProtocolFeatures::REPLY_ACK
+            | VhostUserProtocolFeatures::INBAND_NOTIFICATIONS
```

Comment:
You can remove this one, as it's definitely not supported.

Reply:
Done. I think I mixed up `VHOST_USER_VRING_KICK` and `VHOST_USER_SET_VRING_KICK`.